.travis.yml

```diff
@@ -1,37 +1,54 @@
 # Based on http://conda.pydata.org/docs/travis.html
-language: python
+language: minimal
+
 sudo: false # use container based build
+
 notifications:
   email: false
 
-python:
-  - 2.7
-  - 3.5
-  - 3.6
+env:
+  - PYTHON_VERSION=2.7
+  - PYTHON_VERSION=3.6
+  - PYTHON_VERSION=3.7
 
 matrix:
   include:
-    - python: 3.6
-      env: BUILD_DOCS="true"
+    - env: PYTHON_VERSION=3.7 BUILD_DOCS="true"
 
 before_install:
-  - wget http://bit.ly/miniconda -O miniconda.sh
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda update conda --yes --all
-  - conda config --add channels conda-forge --force
-  - conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
-  - source activate TEST
-  - if [[ $BUILD_DOCS == "true" ]]; then
+  # Build the conda testing environment.
+  - >
+    echo "Installing miniconda and cftime dependencies...";
+    wget http://bit.ly/miniconda -O miniconda.sh;
+    bash miniconda.sh -b -p $HOME/miniconda;
+    export PATH="${HOME}/miniconda/bin:${PATH}";
+    conda update conda --yes --all;
+    conda config --add channels conda-forge --force;
+    export ENV_NAME="test-environment";
+    conda create --yes -n ${ENV_NAME} python=${PYTHON_VERSION} --file requirements.txt --file requirements-dev.txt;
+    source activate ${ENV_NAME};
+    if [[ "${BUILD_DOCS}" == "true" ]]; then
       conda install -qy sphinx;
     fi;
 
-# Test source distribution.
+  # Output environment details.
+  - >
+    conda list -n ${ENV_NAME};
+    conda list -n ${ENV_NAME} --explicit;
```

Comment:
The `conda list -n ${ENV_NAME} --explicit` is for developers to allow them to easily recreate the travis-ci conda environment locally.